.travis.yml

```diff
@@ -1,20 +1,40 @@
-dist: trusty
-sudo: true
-group: beta
 language: scala
 scala:
-- 2.11.11
-- 2.12.3
+  - 2.12.7
 jdk:
-- oraclejdk8
+  - oraclejdk8
+  - openjdk11
+env:
+  matrix:
+    - SCRIPT=scripts/test-sbt
+#   - SCRIPT=scripts/test-gradle
```

Comment:
Keep this running since it is allowed to fail.

Reply:
The `scripts/test-gradle` doesn't exist. It's a different type of failure but I guess we can live with that and have identical `.travis.yml` in exchange. 👍

Reply:
so, should we merge this or not?